Automattic/delphin
# Domain Setup: Add new Connecting Existing Blog page

This pull request adds a new page to the domain setup flow that is displayed to users while connecting their domain to an existing WordPress.com blog:

![screenshot](https://cloud.githubusercontent.com/assets/594356/19241345/7d5c00fa-8f0e-11e6-89a6-537a0eb9302d.png)
#### Testing instructions
1. Run `git checkout add/connecting-existing-blog-page` and start your server, or open a [live branch](https://delphin.live/?branch=add/connecting-existing-blog-page)
2. Open the [`Login` page](http://delphin.localhost:1337/log-in) and connect with an account with domains purchased
3. Navigate to the `My Domains` page
4. Click the `Set Up Now` link on any domain.

At this point, there are two branches of instructions based on whether you're testing a new or existing site.

_Testing an existing site_
5. Select `A blog I've already created` in the next screen and hit the `Next` button
6. Enter any type of url (e.g. `https://myblog.com/bla/bla.bla`) and click the `Next` button
7. Check that you see the host name (e.g. `myblog.com`) in the main heading
8. Click the `Yes, Connect Now` button
9. Assert that you see a page with a message that reads `Just a moment while we set up [.blog domain] to work with your WordPress.com blog at [existing domain].`

_Testing a new site_
5. Select `A blog I'm going to create` in the next screen and hit the `Next` button
6. Select the radio button by the WordPress.com option and submit.
7. Assert that you see a page with a message that reads `Just a moment while we set up [.blog domain] to work with WordPress.com.`
#### Additional notes

This new page is pretty bare bone for now since the design is still been worked on. This pull request also fixes a few page titles.
#### Reviews

@Automattic/sdev-feed
